platform/analytics/google-analytics/build-process-measuring-pageviews-and-events.md

```diff
@@ -44,25 +44,68 @@ To support our React-based single page apps, we have turned on reporting of Hist
+We also send custom event tracking via JavaScript to report on user actions like downloads, successful or failed form submissions, _etc_. Since Event tracking is more customized than the pageview tracking, it relies on defining and instrumenting a dataLayer event. We are also able to send other non-PII/PHI values like `errorKeys` into Google Analytics for further measurement.
 
-During the Discovery phase, your VFS team should have worked with Analytics-Insights to define the key performance indicators you'd like to track. In the build phase, we will work with you to define these requirements for Event Tracking. 
+During the Discovery phase, your VFS team should have worked with Analytics-Insights to define the key performance indicators you'd like to track. In the build phase, we will work with you to define these requirements for Event Tracking. The good news is that many interactions automatically generate Google Analytics events:
 
-_Example:_
 
-Your VFS team is building a new feature for the Profile page and wants to track how many times users submitted an information change for this section. You would structure this event like `window.dataLayer.push({ event: 'profile-transaction'})`. Within Google Tag Manager, we would define a more readable format for an Event Category of "Transactions", Event Action of "Profile Saved", and an Event Label of "profile-transaction". 
+#### Component Library
 
+The component-library is an npm package of reusable components used on va.gov.  Many of these components are programmed to emit a CustomEvent during certain interactions.  va.gov listens for these events and pushes a corresponding event to the dataLayer, which then triggers an event in Google Tag Manager.
 
-_Instrumentation:_
+For example, the AlertBox component calls the dispatchAnalyticsEvent helper function, which emits a CustomEvent on the document.body element.   The listener on va.gov matches the componentName and action in the analyticsEvents lookup table to get an event like  'nav-alert-box-link-click'.  The rest of the fields in the CustomEvent's details are given a prefix of the componentName (kebab format) and sent along with the event. 
+
+##### Other important things:
+
+*   Some components are opt-out and you need to add a disableAnalytics param to the component to disable analytics.
+*   Others (specifically form controls) are opt-in.  Due to the risk of PII/PHI, you must add an enableAnalytics flag to these components to get the CustomEvent to emit.
+*   We try to match the names of the additional params to their component counterparts.  this.props.label would be label while this.props.options label would be optionLabel.
+*   There's a lot of boilerplate in the unit tests for these in the component-library, so use the helper function if possible.
+*   Helpful links:
```

Comment:
@bsmartin-ep This is excellent - would you mind giving this section with the important links a subheader markdown 

```suggestion
*   ### Helpful links:
```